riscv/processor.h

```diff
@@ -272,8 +272,8 @@ class processor_t : public abstract_device_t
   void reset();
   void step(size_t n); // run for n cycles
   void set_csr(int which, reg_t val);
-  reg_t get_csr(int which, insn_t insn, bool write);
-  reg_t get_csr(int which) { return get_csr(which, insn_t(0), false); }
+  reg_t get_csr(int which, insn_t insn, bool write, bool peek = 0);
+  reg_t get_csr(int which) { return get_csr(which, insn_t(0), false, false); }
```

Comment:
change this one to pass `true`.  My intent was that `get_csr(int)` is the "peek" form that skips permission checks.

Reply:
has been modified in push